### PR TITLE
Imx6ullevk FIT bootscr

### PR DESCRIPTION
**Pull request by igoropaniuk**

### CODE REVIEW

**Review by Tim-Anderson — COMMENTED** [review]
Am I missing where the boot.itb is added to the u-boot.itb image?

**Comment by igoropaniuk** [comment]
@Tim-Anderson 
```commit 572c1e0212799733c575b90e9c7934f7950dc2be
Author: Igor Opaniuk <igor.opaniuk@foundries.io>
Date:   Fri Apr 9 17:34:52 2021 +0300

    uboot-fitimage: include boot script in u-boot.itb
    
    Signed-off-by: Igor Opaniuk <igor.opaniuk@foundries.io>
```

**Review by Tim-Anderson — APPROVED** [review]
LGTM

File: meta-lmp-base/classes/uboot-fitimage.bbclass [hunk 1]
```diff
@@ -208,6 +208,8 @@ do_deploy_prepend() {
 	if ${ATF_SUPPORT}; then
 		ATF_ELF="${DEPLOY_DIR_IMAGE}/$(basename -s .bin ${ATF_BINARY}).elf"
 		ATF_LOAD_ADDR=$(${READELF} -h ${ATF_ELF} | egrep -m 1 -i "entry point.*?0x" | sed -r 's/.*?(0x.*?)/\1/g')
+	else
+		ATF_LOAD_ADDR="0x0"
```
Comment (ricardosalveti) [inline meta-lmp-base/classes/uboot-fitimage.bbclass:212]: Why do we need to set it to 0x0 when not used? Not clear from the commit log.

Comment (igoropaniuk) [reply meta-lmp-base/classes/uboot-fitimage.bbclass:212]: Added explanation

**Comment by Tim-Anderson** [comment]
Maybe this should be part of PR #312

**Comment by ricardosalveti** [comment]
> Maybe this should be part of PR #312

This should be fine to land independently, as we did for imx8 (mostly because this pr is useful on its own anyway).

File: meta-lmp-base/classes/uboot-fitimage.bbclass (hunk 1 above)
Comment (ricardosalveti) [reply meta-lmp-base/classes/uboot-fitimage.bbclass:212]: Thanks!

**Review by ricardosalveti — APPROVED** [review]
LGTM

**Pull request merged**